### PR TITLE
Added base navbar for live.

### PR DESCRIPTION
**Pull request by AmanGotchu**

### CODE REVIEW

File: src/Pages/Live/LiveSchedule/LiveSchedule.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import React from "react";
```
Comment (jay-khatri) [inline src/Pages/Live/LiveSchedule/LiveSchedule.js]: Nice!

File: src/Pages/PageRouter.js [hunk 2]
```diff
@@ -4,21 +4,39 @@ import Landing from "Pages/Landing/Landing";
...
+      pathName = pathName.substring(0, pathName.length - 1);
+    }
+    if (subdomain === "live") {
+      switch (pathName) {
+        case "":
+          return <LiveWrapper />;
+        case "/schedule":
+          return <LiveScheduleWrapper />;
+        case "/maps":
+          return <LiveMaps />;
```
Comment (jay-khatri) [inline src/Pages/PageRouter.js]: Don't u need to render `LiveMapsWrapper` and `LivePrizesWrapper` here? Can we disclude these lines for the meantime?

File: src/Pages/Live/LiveSchedule/LiveSchedule.json [hunk 3]
```diff
@@ -0,0 +1,159 @@
+{
```
Comment (jay-khatri) [inline src/Pages/Live/LiveSchedule/LiveSchedule.json]: spacing is really funky but we can fix that later

Comment (AmanGotchu) [reply src/Pages/Live/LiveSchedule/LiveSchedule.json]: Oh lmao I didn't notice :(

File: src/Pages/PageRouter.js (hunk 2 above)
Comment (AmanGotchu) [reply src/Pages/PageRouter.js]: Yes. I did fix these in two other branches (for maps and prizes) but I'll remove it in this one.

File: src/Pages/Live/Live.js [hunk 4]
```diff
@@ -1,11 +1,20 @@
 import React from "react";
 import { Background } from "Pages/Events/EventsStyles";
+import LiveNavBar from "Pages/Live/LiveNavbar/LiveNavbar";
 
 class Live extends React.Component {
+  state = { hello: "hello" };
```
[the remark this answers is not in the document]
Comment (AmanGotchu) [reply src/Pages/Live/Live.js]: Done

**Pull request merged**